proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,72 @@ The Working Group identified four possible workflows for event passing, based on
 * Collection.  ("Give me all your things, that I may do something with that list.")
 * Alternative chain.  ("Here's a thing; the first one of you that can handle it do so, then stop.")
 
-On further review, it was determined that Collection was a special case of Object enhancement (the collection being the object that is enhanced).  Alternative chain is similarly a special case of Object enhancement, as the signature is identical and the dispatch workflow is nearly identical.  That leaves two relevant workflows:
+On further review, it was determined that:
 
-* Notification
-* Modification
+* Collection was a special case of Object enhancement (the collection being the object that is enhanced)
+* Alternative chain is similarly a special case of Object enhancement, as the signature is identical and the dispatch workflow is nearly identical, albeit with an extra check included.
+* One-way notification is a degenerate case of the others, or can be represented as such.
 
-Notification can safely be done asynchronously (including delaying it through a queue) but Modification by nature involve passing data back to the caller and thus must be synchronous.  Despite that difference the Working Group determined that the use cases were close enough to be considered in a single PSR.  The two different workflows however are represented by two different interfaces, `MessageNotifierInterface` (for Messages) and `TaskProcessorInterface` (for Tasks).
+Although in concept one-way notification can be done asynchronously (including delaying it through a queue), in practice few explicit implementations of that model exist.  That provides fewer places from which to draw guidance on details such as proper error handling.  After much consideration the Working Group elected to not provide an explicitly separate workflow for one-way notification as it could be adequately represented as a degenerate case of the others.
 
-### 4.2 Intended use cases
+### 4.2 Example applications
```

Comment:
I'm not entirely happy with this list where it is, but it was in the spec previously to differentiate tasks from messages and I didn't want to just lose it.  I'm open to suggestions on where to move it or how to modify it.